client/state/sharing/publicize/reducer.js

```diff
@@ -37,9 +43,11 @@ export function fetchingConnections( state = {}, action ) {
 		case SERIALIZE:
 			return state;
 		case DESERIALIZE:
-			return state;
+			if ( isValidStateWithSchema( state, fetchingConnectionsSchema ) ) {
```

Comment:
:+1: I'd be fine with not persisting this. If we choose not to persist this, I think I can ditch the top level publicize schema.